word/views_api.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from fuzzywuzzy import fuzz
+from fuzzywuzzy import process
+from .models import Word, Description
+
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+
+
+@csrf_exempt
+@api_view(['GET'])
+@authentication_classes([])
+@permission_classes([])
+def similar_words(request):
+    if request.GET:
+        base_word = request.GET.get("word")
+        if base_word:
+            list_of_words = list(Word.objects.all().values_list("word", flat=True))
```

Comment:
This should be filtered by the language the new word is for

Reply:
ok